### PR TITLE
Update for Swift 3

### PR DESCRIPTION
**Pull request by EdenShapiro**
These changes look okay? @GabrielleM (CCing Ayush in case you don't have time to look them over) @saraswatayu

### CODE REVIEW

File: Alien Adventure/Alien.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ class Alien: SKSpriteNode {
         return requests[0]
     }
     var getNextRequest: UDRequest? {
-        if ++currentRequestIndex < requests.count {
+        if (currentRequestIndex+1) < requests.count {
```
Comment (saraswatayu) [inline Alien Adventure/Alien.swift]: ++currentRequestIndex also increments currentRequestIndex -- currentRequestIndex + 1 does not

**Comment by saraswatayu** [comment]
Can you squash all of these commits into one?

**Comment by EdenShapiro** [comment]
Better? @saraswatayu

File: Alien Adventure/Alien.swift [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Alien: SKSpriteNode {
         return requests[0]
     }
     var getNextRequest: UDRequest? {
-        if ++currentRequestIndex < requests.count {
+        if Int(currentRequestIndex += 1) < requests.count {
```
Comment (saraswatayu) [inline Alien Adventure/Alien.swift]: I would suggest breaking this up into two lines:
`currentRequestIndex += 1`
`if currentRequestIndex < requests.count {`

**Comment by saraswatayu** [comment]
I'm not sure combining the commits worked. Try using a rebase. I don't have a good explanation for this but here's a tutorial: http://code.tutsplus.com/tutorials/rewriting-history-with-git-rebase--cms-23191. Section 2 is most pertinent. You'll probably need to force push this to your branch.

**Pull request closed without merging**